File: Day_Three/Afternoon-session/java/MiniParkingSystem.java
import java.util.Arrays;

public class MiniParkingSystem{

    public static int[] getSlot(int[] slots, int spaced){
        int space = spaced - 1;
        
         int[] array = new int[slots.length];
        
        

            if(slots[space] == 0){
                array[space] = 1;
            }
        
        return array;
    }

    public static int[] removeSlot(int[] slots, int spaced){
        int space = spaced - 1;
        
         int[] array = new int[slots.length];
        
        

            if(slots[space] == 0){
                array[space] -= slots[space];
            }
        
        return array;
    }

    

    

    

    

}

File: Day_Three/Afternoon-session/python/mini_parking_system.py



def getSlot(slots, spaced):
#    slots = list(slots)

    space = spaced - 1
    len(slots) == 20;
    array = len(slots)
    
    if(slots[space] == 0):
        array[space] = 1

    return array

slots = 20

spaced = 5
print(getSlot(slots, spaced))

 
         
